### PR TITLE
Add Group Model

### PR DESCRIPTION
**Pull request by AgustinOsorio**

### CODE REVIEW

File: src/migrations/20200303162506-create-group-model.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+      },
+      type: {
+        type: Sequelize.ENUM,
+        values: [
+          'PAIR',
+          'GROUP'
+        ],
+        allowNull: false
+      },
+      deadline: {
```
Comment (abelosorio) [inline src/migrations/20200303162506-create-group-model.js]: Let's rename this to "expiresAt".

File: src/models/group.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    },
+    type: {
+      type: DataTypes.ENUM,
+      values: [
+        'PAIR',
+        'GROUP',
+      ],
+      allowNull: false
+    },
+    ExpiresAt: {
```
Comment (abelosorio) [inline src/models/group.js]: You have a capital E.

File: src/models/group.js [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+      type: DataTypes.DATE,
+      allowNull: false
+    },
```
Comment (abelosorio) [inline src/models/group.js]: Trailing comma here.

File: src/models/product.js [hunk 4]
```diff
@@ -1,27 +1,31 @@
...
+    },
+    name: {
+      type: DataTypes.STRING,
+      allowNull: false
+    },
+    description: {
+      type: DataTypes.STRING,
+      allowNull: true
+    },
+    price: {
```
Comment (abelosorio) [inline src/models/product.js]: With the addition of the group, the price will belong to the group. The market price will still belong to the product.

File: src/migrations/20200303162506-create-group-model.js (hunk 1 above)
Comment (AgustinOsorio) [reply src/migrations/20200303162506-create-group-model.js]: By mistake I created the expiresAt but forgot to delete this, already fixed.

File: src/models/group.js (hunk 2 above)
Comment (AgustinOsorio) [reply src/models/group.js]: Fixed

File: src/models/group.js (hunk 3 above)
Comment (AgustinOsorio) [reply src/models/group.js]: Fixed

File: src/models/product.js (hunk 4 above)
Comment (AgustinOsorio) [reply src/models/product.js]: I created the field 'discount' in Group model to deduct the discounted price of the group in the backend. Check it please.

File: src/migrations/20200303162506-create-group-model.js [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+      type: {
+        type: Sequelize.ENUM,
+        values: [
+          'PAIR',
+          'GROUP'
+        ],
+        allowNull: false
+      },
+      discount: {
+        type: Sequelize.FLOAT,
```
Comment (abelosorio) [inline src/migrations/20200303162506-create-group-model.js]: Let's use integer for discounts, for simplicity.

File: src/models/group.js [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+    discount: {
+      type: DataTypes.FLOAT,
+      allowNull: false
+    },
+    expiresAt: {
+      type: DataTypes.DATE,
+      allowNull: false
+    }
```
Comment (abelosorio) [inline src/models/group.js]: Let's add "minParticipants" and "maxParticipants". max is not required.

Comment (AgustinOsorio) [reply src/models/group.js]: Ok Done

File: src/migrations/20200303162506-create-group-model.js (hunk 5 above)
Comment (AgustinOsorio) [reply src/migrations/20200303162506-create-group-model.js]: Done

**Pull request merged**